scipy/interpolate/interpolate.py

```diff
@@ -22,12 +22,13 @@
 from ._bsplines import make_interp_spline, BSpline
 
 
-def lagrange(x, w):
+def lagrange(x, y, reduce=True):
```

Comment:
Can't just rename arguments like this : it's breaking back compat for anybody who uses a keywords arg, `lagrage(x, w=...)`.